src/simulations/action_translator.py

```diff
@@ -307,6 +325,16 @@ def actions_to_env_vars(self, loads, home_vars, action, date, time_step):
                     + charge - discharge + res['l_ch'] \
                     - home_vars['gen'][home]
 
+                if self.reactive_power_for_voltage_control:
+                    # based on flexible store actions, calculate flexible q_car action
+                    # reactive power battery between -1 and 1 where
+                    # -1 max export
+                    # 1 max import
+                    res['q'] = flexible_q_car_action \
+                        * np.sqrt(self.max_apparent_power_car**2 - res['ds']**2)
+                    flexible_q_car[home] = res['q']
```

Comment:
be careful that `ds` (change in storage levels) `!=p_car_flex`

Reply:
Should this be changed to `charge - discharge + res['l_ch']` to also include the losses?

Reply:
yes but I am now also working on this in parallel so you can hold back for now. (I cannot reproduce the errors in Notion for now because I get other errors such as with the max apparent power etc)

Reply:
Yes, I noticed these ones too. I think they appeared because of the newly added `eta_ch` to some of the active power calculations. Be careful that this here is another branch (passive homes) that I had created off of the q_car branch. Let me know when I can push again :)

Reply:
oh so this is not the latest version? am I duplicating work if working here?

Reply:
do you want to merge things, fix this, etc, and when you have the errors in notion again I can have a look?

Reply:
If you started working on this branch, we can then merge it again with the q_car one. The latest version is actually the one you reviewed before.

Reply:
Sure

Reply:
my bad, I was actually working on `decision_reactive_power_battery`, but commented on the wrong one. The same comment applies there :)

Reply:
I'll close this conversation here as this has been resolved in #55